opennms-model/src/main/java/org/opennms/netmgt/model/OnmsSnmpInterface.java

```diff
@@ -527,7 +527,6 @@ public boolean isPollEnabled() {
     @XmlElement(name="nodeId")
     //@XmlIDREF
     @XmlJavaTypeAdapter(NodeIdAdapter.class)
-    @JsonIgnore
```

Comment:
Wouldn't removing this cause the entire node to be JSON-encoded? It wouldn't cause any circular issues because OnmsSnmpInterfaces are annotated with JsonIgnore within OnmsNode, but the XML and JSON encodings would be different. Is that an issue?

Reply:
We're using `JaxbAnnotationIntrospector` which makes jackson also process the XML annotations. Therefore the adapter defined above should break the loop. But I have not tested this for this particular use case.